events/event_handler.go

```diff
@@ -160,6 +161,18 @@ func (fh *eventHandler) handleStatusEvent(body []byte) error {
 		"TaskStatus": task.TaskStatus,
 	}).Info("Got StatusEvent")
 
+	app, err := fh.marathon.App(task.AppID)
```

Comment:
I checked the code and we already did it. IMO best to change error and check if this is real error. Again if we want to deregister a service that is not registered this is not a problem. The problem was before, why we haven't registered it if we should?